### ISSUE TITLE
Adding cssnext alongside sass, less and stylus

### ISSUE THREAD

**Issue by alexandruradulescu** [iamvdo/pleeease#66]
Hi,

First thing I want to mention is that I know there was a discussion about removing the pleeease.next module and the whole so called future css syntax.
I want to suggest adding cssnext to pleeease-io just as there is sass, less, stylus.

Secondly removing pleeease.next make a lot of sense, not doing duplicate work.
But I have to say I love the whole implementation of cssnext, not for the reason of "future syntax" but simply because it is a simpler, more back to basics way of writing css. I think that less and sass are too powerful and can end up making very bad code if you're not very very good.

Thirdly, as I understand, pleeease.io idea is to make it simple and powerful even for not so experienced developers to use css processors (pre, post etc). As my personal experience was that I never managed to get cssnext working perfectly repeatedly, I think this would be a great addition to pleeease.io.

So in conclusion, as cssnext is quite a famous and used css processor, with a good implementation, I would love if you could add it to pleeease.io.

Thanks :-)
